### ISSUE TITLE
Dropbox connection not showing all files

### ISSUE THREAD

**Issue by mike4644** [gogolith/gocoeditserver#91]
On iPhone 7 Plus, when opening files from Dropbox I am not seeing all of the files that exist in the dropbox folder. If I open the Dropbox app, I do see all of the files.

**Comment by flaxton** [comment]
This has been driving me crazy. A lot of folders won’t show on GoCoEdit, but they show on Files or Dropbox or other apps that use Files or Dropbox.

**Comment by mike4644** [comment]
Sadly, I had to find a different app to use because of this one issue.

**Comment by flaxton** [comment]
I punted and created a repository here on GitHub, and then I'm using Working Copy on iPadOS, to keep everything in sync. Then using git command line on the Mac and now I have the files easy to access ;-)

**Comment by Drag0nsilver** [comment]
This issue affects me as well. iOS 14.7.1 on iPhone 12 Pro, as well as iPadOS 14.7.1 on iPad 6th gen. Are there plans to address this?

**Comment by iandallas1978** [comment]
I'm seeing *most* files show up through the Dropbox connection, but a large number of files in the root folder aren't showing up. 

There doesn't seem to be any pattern to which files are visible and which aren't. They all used to be visible in earlier versions of the app, and are currently visible in both the Dropbox app and in other Dropbox-connected apps (on iPadOS 14 + 15)

**Comment by ryan-p-randall** [comment]
Unfortunately, I'm still seeing this as well. I can see far, far more files in the folders I use for markdown notes in my Dropbox app than I can in the same folder in GoCoEdit. I have about 700 notes in one folder, but I can only see about 100 of them in GoCoEdit.  

The only potential pattern with my files is that maybe GoCoEdit has trouble handling files with longer file names.  

GoCoEdit is one of my favorite editors on iOS, so I hope this will be addressed soon.

**Comment by RichardBJ** [comment]
Yes, help with this issue would be great.  The app is unusable for me as it stands; most folders in my Dropbox root show up, but a percentage (20%) of folders do not.  I can not see what the pattern is. Possibly the ones I need don't show up.  that could actually be causal of course (they have changed state for example).  Certainly not folder name length in my case.  My root folder "Python", where obviously all my Python code sits is not appearing.  Old obsolete folders seem fine...
